src/main/java/org/opentripplanner/standalone/config/framework/json/ParameterBuilder.java

```diff
@@ -652,4 +652,8 @@ private OtpAppException error(String message) {
   private OtpAppException error(String message, Exception e) {
     return target.createException(message, paramName(), e);
   }
+
+  private enum DefaultValue {
+    UNDEFINED,
+  }
```

Comment:
I think I would use a constant for this instead omitting defining a new unused type:

```
private static final Object UNDEFINED = new Object();
```

Reply:
I didn't think of that, I'll change it so it's a bit simpler.